content/1.index.md

```diff
@@ -2,6 +2,9 @@
 
 [Gevamu* Payments Solution](https://gevamu.com/), developed by [Exactpro](https://exactpro.com/) based on [R3 Corda](https://www.r3.com/products/corda/), is aimed at fund distribution between a payer (the :term[Participant]{term="Participant"}) and a :term[payment service provider]{term="PSP"} (PSP). 
 
+---
+\*from Sinhala: ගෙවමු – *Let’s pay*
+
```

Comment:
I think footer from paper docs won't work in case of website. It is better to place this information right after the word. Just like it is done in other docs: https://vitejs.dev/guide/

```suggestion
[Gevamu (from Sinhala: ගෙවමු – *Let’s pay*) Payments Solution](https://gevamu.com/), developed by [Exactpro](https://exactpro.com/) based on [R3 Corda](https://www.r3.com/products/corda/), is aimed at fund distribution between a payer (the :term[Participant]{term="Participant"}) and a :term[payment service provider]{term="PSP"} (PSP). 

```